3021.bugfix

```diff
@@ -0,0 +1 @@
+In tests, use stronger password.  [maurits]
```

Comment:
Is that a path typo, or done in purpose? :thinking: this file should go to the `news` folder, isn't it?

Reply:
Oops. :-) Fixed.